package.json

```diff
@@ -85,8 +85,5 @@
     "rollup": "^3.23.0",
     "turbo": "1.7.4",
     "typescript": "4.9.3"
-  },
-  "devDependencies": {
-    "migrate-twig-helper": "workspace:^"
```

Comment:
Was this package not referenced by any other packages?

Reply:
Nope, just this one.